Apps/TwoEleven/manifest.properties

```diff
@@ -0,0 +1,10 @@
+[manifest]
+version=0.1
+[target]
+sdk=0.7.0-SNAPSHOT2
+platforms=esp32,esp32s3
+[app]
+id=two.shadowtrance.twoeleven
```

Comment:
Would you mind to change it to `one.tactility.twoeleven`? (all the official apps have the `one.tactility.` prefix)